torch/_dynamo/config.py

```diff
@@ -370,6 +372,27 @@ def default_debug_dir_root():
 # If to log Dynamo compilation metrics into log files (for OSS) and Scuba tables (for fbcode).
 log_compilation_metrics = True
 
+# Reorders logging statements (prints/warnings) to construct larger graphs, as
+# dynamo will by default graph break on these. Note that there are some
+# limitations to this, such as how it does not correctly print objects that were
+# mutated after the print statement. If using a custom logging function, you can
+# add the function to `reorderable_logging_functions` to allow it to be
+# reordered.
+reorder_logs = False
+
+# A set of logging functions which can be reordered to the end of graph breaks
+reorderable_logging_functions = {
+    logging.critical,
+    logging.debug,
+    logging.error,
+    logging.exception,
+    logging.info,
+    logging.log,
+    logging.warning,
+    print,
+    warnings.warn,
+}
+
```

Comment:
I think probably one flag is better then two, should we only keep the ```reorderable_logging_functions``` and None by default? Users can set it if they want to reorder any function rather than graph break on it.